server/src/interfaces/person.interface.ts

```diff
@@ -35,20 +40,26 @@ export interface PeopleStatistics {
   hidden: number;
 }
 
+export interface DeleteAllFacesOptions {
+  sourceType?: string | null;
```

Comment:
when you pass `null` that deletes non exif sourced data? I think it would be better to have an actual value, it might be too easy to accidentally send null imo.

Reply:
Hi @jrasm91, @mertalev. Would it be acceptable to remove all faces when `sourceType` is undefined (like the current one drafted at https://github.com/immich-app/immich/commit/2f2e288e01cabf285d48464403fef939b13e73a7#diff-bdd10fc89e503edc0aa770142c760f507c6b6817b12b1b3a28df3e7d7145ca69R54-R56) or should we add an `all` attribute to `DeleteAllFacesOptions` to make it more clear?

Reply:
I think if no option (omitted) removes everything, that is fine.

Reply:
That's correct